raft/node.go

```diff
@@ -38,6 +38,7 @@ var (
 // SoftState provides state that is useful for logging and debugging.
 // The state is volatile and does not need to be persisted to the WAL.
 type SoftState struct {
+	// Lead MUST use atomic operation to access, keep 64bit aligned.
```

Comment:
I don't think this is a good godoc since it doesn't describe what `Lead` does.

`Lead uint64 // must use atomic operations to access; keep 64-bit aligned.`